t/t3428-rebase-edit-todo.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test_description='rebase should reread the todo file if an exec modifies it'
+
+. ./test-lib.sh
+
+todo=$(git rev-parse --git-dir)/rebase-merge/git-rebase-todo
```

Comment:
You don't need double quotes here even with the spaces, because variable assignments are special in bourne shell, and do not go through the extra round of whitespace splitting that command arguments do. So:

```
foo=$(echo look at all these spaces)/spaces_here_would_turn_this_into_a_command
```

is fine. The quotes don't hurt, of course (and I agree that just writing `.git` is fine for the test harness).